lib/main.dart

```diff
@@ -38,7 +37,6 @@ GetIt getIt = GetIt.instance;
 
 void instantiateServices() {
   getIt.registerSingleton(UserService());
-  getIt.registerSingleton(AnalyticsService());
```

Comment:
You have to use GetIt in order to use Analytics. Put back Analytics Service and use it instead of Firebase analytics directly